### PR TITLE
feat: browser deeplink

### PR DESCRIPTION
**Pull request by iGroza**
Fixes #

## Proposed Changes

- handle deeplink to open browser

https://github.com/haqq-network/haqq-wallet/assets/25716589/5c511860-c80b-4983-b450-7ae016f77336

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

**Comment by iGroza** [comment]
wait for @VorobevSA approve

File: src/event-actions/on-deep-link.ts [hunk 1]
```diff
@@ -62,6 +63,20 @@ export async function onDeepLink(
   if (link.startsWith(`${ValidUrlProtocol.haqq}:`)) {
     let params = link.split(':');
 
+    if (link.startsWith('haqq://browser')) {
```
Comment (devkudasov) [inline src/event-actions/on-deep-link.ts:66]: Make sense to move it into const

File: src/event-actions/on-deep-link.ts [hunk 2]
```diff
@@ -62,6 +63,20 @@ export async function onDeepLink(
...
+      const uri = decodeURIComponent(link.replace('haqq://browser?uri=', ''));
+      openInAppBrowser(uri);
+      return true;
+    }
+
+    if (link.startsWith('haqq://web3browser')) {
```
Comment (devkudasov) [inline src/event-actions/on-deep-link.ts:72]: You can use const here as well

File: src/event-actions/on-deep-link.ts [hunk 3]
```diff
@@ -62,6 +63,20 @@ export async function onDeepLink(
...
+      const uri = decodeURIComponent(
+        link.replace('haqq://web3browser?uri=', ''),
+      );
+      openWeb3Browser(uri);
+      return true;
+    }
+
     if (link.startsWith('haqq://wc')) {
```
Comment (devkudasov) [inline src/event-actions/on-deep-link.ts:80]: Many different url prefixes. Maybe better use enum instead of consts?

**Review by devkudasov — CHANGES_REQUESTED** [review]

**Review by devkudasov — APPROVED** [review]

**Pull request merged**